```
Cannot find module 'process'
Error: Cannot find module 'process'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Class.module.exports.run (/Users/jeremy/Desktop/throwAway/sl-ember-components/node_modules/ember-cli-jsdoc/lib/commands/ember-cli-jsdoc.js:10:19)
    at Class.<anonymous> (/Users/jeremy/Desktop/throwAway/sl-ember-components/node_modules/ember-cli/lib/models/command.js:146:17)
    at lib$rsvp$$internal$$tryCatch (/Users/jeremy/Desktop/throwAway/sl-ember-components/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/jeremy/Desktop/throwAway/sl-ember-components/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (/Users/jeremy/Desktop/throwAway/sl-ember-components/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (/Users/jeremy/Desktop/throwAway/sl-ember-components/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1198:9)
```

This was encountered when clone `sl-ember-components`, changing the `ember-cli-jsdoc` entry in _package.json_ to `spikedkira/ember-cli-jsdoc#issue-16`, running "nombom" and then `ember ember-cli-jsdoc`.  Need to add it as a dependency?

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/softlayer/ember-cli-jsdoc/20#-:-KCXmpQ0dCx-MgeqNK7d:1055573694)_

<!-- Sent from Reviewable.io -->

--- a/lib/commands/ember-cli-jsdoc.js
+++ b/lib/commands/ember-cli-jsdoc.js
@@ -7,12 +7,11 @@ module.exports = {
         var exec = require( 'child_process' ).exec;
         var rsvp = require( 'rsvp' );
         var path = require( 'path' );
-        var cwd = require( 'process' ).cwd;
         var cmdPath = path.join( 'node_modules', 'ember-cli-jsdoc', 'node_modules', '.bin', 'jsdoc' );
         var chalk = require( 'chalk' );
 
         return new rsvp.Promise( function( resolve, reject ) {
-            exec( cmdPath + ' -c jsdoc.json', { cwd: cwd() }, function( error, stdout, stderr ) {
+            exec( cmdPath + ' -c jsdoc.json', { cwd: process.cwd() }, function( error, stdout, stderr ) {
                 console.log( stderr );
 
                 if ( error ) {

